ec_api/apps/users/forms.py

```diff
@@ -19,3 +20,16 @@ def clean_email(self):
         """
         email = self.cleaned_data["email"]
         return User.objects.normalize_email(email)
+
+
+class APIKeyForm(forms.ModelForm):
+    class Meta:
+        model = APIKey
+        fields = ["name", "usage_reason"]
+        widgets = {
+            "user": forms.HiddenInput,
```

Comment:
This is a bit of a small point that we can leave for now, but I've realised that there isn't a standard pattern for saving the current user who submitted a form.

Ideally the backend would get the user from the session, rather than POSTing it from the client. It's not important in this case, but consider that a user could change the form value here to create a key for a different user.

Reply:
Yeah I did realise this is a bit of a vulnerability this morning, but as it happens was planning to update anyway. So now the user is set in the `form_valid` method on the view, using the logged in user from the request. Which I think is a much cleaner anyway, as it uses the django ModelForm methods directly